CHANGELOG.rst

```diff
@@ -61,6 +61,11 @@ General
   :ref:`debops.global_handlers` role to allow for easier cross-role handler
   notification. The role has been imported in roles that rely on the handlers.
 
+:ref:`debops.netbox` role
+'''''''''''''''''''''''''
+
+- Update to Netbox 2.8. The newly added plugin support can be configured from DebOps.
```

Comment:
The update notice should be moved to the "Updates of upstream application versions" section in the Changelog. It's currently present if it hasn't been there before. Fixing this will also fix the merge conflict.